index.html

```diff
@@ -12956,7 +12956,42 @@ <h2>A project for first-time contributions</h2>
             </div>
             <!-- ________ Contributor's card END ________  -->
 	
-
+            <!-- ________ Zach's card START ________  -->
+            <div class="card">
+                <p class="name">Zachary King</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="YOUR twitter account link" target="_blank">@Zach_A_King</a>
+                </p>
+                <p class="about">
+                    I am an aspiring Software Engineer, and I am getting closer one line of code at a time.
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://www.twitch.tv/codinggarden" target="_blank"
+                                title="The Best JavaScript Live Coder to learn from">
+                                Coding Garden
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.codenewbie.org/" target="_blank" title="A wonderful place for those new to coding">
+                                CodeNewbie
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.udemy.com/course/the-web-developer-bootcamp/" target="_blank"
+                                title="The best web development Udemy course of all time">
+                                Colt Steele's "The Web Developer Bootcamp"
+                            </a>
+                        </li>
+                    </ul>
+                    <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
```

Comment:
```suggestion
```
You do not need this comment in your card, You should delete it

Reply:
Ignore this. I just realised we had added the confusing comment to the project. It's been removed now and your PR merged